### ISSUE TITLE
Add MacOS/darwin release builds

### ISSUE THREAD

**Issue by mlange-42** [mlange-42/tiny-world#154]
Was not able to get it to run. Getting errors like these:

```
clang: error: argument unused during compilation: '-arch x86_64' [-Werror,-Wunused-command-line-argument]
```

or

```
# runtime/cgo
gcc_libinit.c: In function '_cgo_try_pthread_create':
gcc_libinit.c:157:17: error: implicit declaration of function 'nanosleep' [-Werror=implicit-function-declaration]
  157 |                 nanosleep(&ts, nil);
      |                 ^~~~~~~~~
cc1.exe: all warnings being treated as errors
```

**Comment by Ecostack** [comment]
Maybe this one can be closed

**Comment by mlange-42** [comment]
Fixed in #165
